taprpc/taprootassets.proto

```diff
@@ -424,6 +443,11 @@ message Asset {
     // determined for script keys that aren't known to the wallet of the local
     // tapd node.
     bool script_key_has_script_path = 19;
+
+    // This field defines a decimal display value that may be present. This is
+    // a wrapper around the uint32 that signals whether the value has been set
+    // or not.
```

Comment:
Add "If this field is null, it means the presence of a decimal display field is unknown in the current context".